freqtrade/rpc/telegram.py

```diff
@@ -146,6 +157,71 @@ def _status(bot: Bot, update: Update) -> None:
 
 
 @authorized_only
+def _status_table(bot: Bot, update: Update) -> None:
+    """
+    Handler for /status table.
+    Returns the current TradeThread status in table format
+    :param bot: telegram bot
+    :param update: message update
+    :return: None
+    """
+    short_version = False
+    params = update.message.text.replace('/status', '').split(' ')
+    if 'short' in params:
+        short_version = True
+
+    # Fetch open trade
+    trades = Trade.query.filter(Trade.is_open.is_(True)).all()
+    if get_state() != State.RUNNING:
+        send_msg('*Status:* `trader is not running`', bot=bot)
+    elif not trades:
+        send_msg('*Status:* `no active order`', bot=bot)
+    else:
+        trades_list = []
+        for trade in trades:
+            # calculate profit and send message to user
+            current_rate = exchange.get_ticker(trade.pair)['bid']
+            current_profit = 100 * ((current_rate - trade.open_rate) / trade.open_rate)
+            orders = exchange.get_open_orders(trade.pair)
+            orders = [o for o in orders if o['id'] == trade.open_order_id]
+            order = orders[0] if orders else None
+
+            fmt_close_profit = '{:.2f}'.format(trade.close_profit) if trade.close_profit else 'No'
+            fmt_current_profit = '{:.2f}'.format(current_profit)
+
+            row = [
+                trade.id,
+                trade.pair,
+                shorten_date(arrow.get(trade.open_date).humanize()),
```

Comment:
I think the `shorten_date` function can be simplified if you pass `only_distance=True` to humanize()